moj-node/assets/javascript/games/breakout/index.js

```diff
@@ -0,0 +1,197 @@
+var gameCanvas = document.getElementById("gameCanvas");
```

Comment:
Just needs the tabbing sorting for formatting :)

Reply:
This initial block may need to be in a function called by onLoad? It's possible that the code runs before the HTML has been generated and thus gameCanvas won't exist?

Reply:
First example here... https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event

Reply:
function applied to window.onload